shared/constants/verification.ts

```diff
@@ -0,0 +1,23 @@
+import { Hex } from '@metamask/utils';
+import {
+  EXPERIENCES_TYPE,
+  FIRST_PARTY_CONTRACT_NAMES,
+} from './first-party-contracts';
+
+export const TX_SIG_LEN = 130;
+export const EXPERIENCES_TO_VERIFY = [EXPERIENCES_TYPE.METAMASK_BRIDGE];
+export const TRUSTED_SIGNERS: Partial<Record<EXPERIENCES_TYPE, Hex>> = {
+  [EXPERIENCES_TYPE.METAMASK_BRIDGE]:
+    '0x533FbF047Ed13C20e263e2576e41c747206d1348',
+};
+
+// look up the corresponding experience and the chain id within FIRST_PARTY_CONTRACT_NAMES
+export const addrToExpMap = Object.entries(FIRST_PARTY_CONTRACT_NAMES).reduce(
```

Comment:
Minor, but even though this is calculated, is this still a constant so uppercase and underscores?

Reply:
I'm not sure I'm following... Do you mean `FIRST_PARTY_CONTRACT_NAMES` or the result of this?

Reply:
Apologies, I just meant should `addrToExpMap` be something like `EXPERIENCE_DATA_BY_ADDRESS`?

Reply:
yeah i think the introduction of getExperience() doesn't make this relevant anymore